package.json

```diff
@@ -24,6 +24,7 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "css-loader": "^0.28.0",
+    "moment": "^2.18.1",
```

Comment:
moment is notoriously big when bundled, it's ok if you ignore all the localized files. [This is what I did for the facilitator dashboard](https://github.com/p2pu/learning-circles/blob/master/webpack.config.js#L54).